### PR TITLE
Feature/hpr006

### PR DESCRIPTION
**Pull request by james-gray**
HPR006

### CODE REVIEW

File: hpr006_works-14-18/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## Usage
+All tracks have been stress-tested in production dancefloor sets over the past decade in various contexts, and are ready for your enjoyment.
+
+1. `Do It Again` is a dreamy 4x4 study, based around swirling DX7 chords, a powerful bassline and crisp 909 drums, with a formant-enhanced sample of Toni Braxton's "You're Makin' Me High."
+
+2. `Hurtin' Thing` is a shuffling ode to the French Touch, a layer cake built around a sample of Gloria Ann Taylor's rendition of Lou Rawls' "Love is a Hurtin' Thing", with a dash of DX7 organ and chimes, and a healthy ration of punchy drums.
+
+3. `Hydroplane City` is a straightforward, tropical nod to the 90s, with melodies and chords using classic presets from both the DX7 and Roland JD-990, and a thick Moog bassline.
+
+4. `Truckee` started its life as a corporate jingle, before being twisted beyond recognition into wonky, filtered house tune inspired by the sounds of the Canadian Riviera.
```
Comment (sabjorn) [inline hpr006_works-14-18/README.md]:
```suggestion
4. `Truckee` started its life as a corporate jingle, before being twisted beyond recognition into a wonky, filtered house tune inspired by the sounds of the Canadian Riviera.
```

or

```suggestion
4. `Truckee` started its life as a corporate jingle, before being twisted beyond recognition into wonky, filter house inspired by the sounds of the Canadian Riviera.
```

**Review by sabjorn — CHANGES_REQUESTED** [review]
one edit suggested

File: hpr006_works-14-18/README.md (hunk 1 above)
Comment (james-gray) [reply hpr006_works-14-18/README.md]: I wouldn't call it "filter house", but I can try to tweak the grammar

**Comment by james-gray** [comment]
@sabjorn updates made 🙌

**Review by sabjorn — APPROVED** [review]

**Pull request merged**